lib/disk.js

```diff
@@ -1,29 +1,44 @@
 'use strict';
 
-const filedisk = require('file-disk');
-
 const constants = require('./constants');
 
-class DiskWrapper extends filedisk.DiskWrapper {
+const promiseToCallback = (promise, callback, transform) => {
+	promise
+	.then((result) => {
+		const args = (transform === undefined) ? [] : transform(result);
```

Comment:
Should this be `transform == null`?

Reply:
I don't think so. When it is called as `promiseToCallback(promise, callback)`, transform is `undefined`.

Reply:
But you'll want to guard against both in case of external consumers changing what they pass in – generally both of those values (undefined, and null) should be treated the same in consuming code

Reply:
This is a helper function that is not exported.
It is only called in 4 places below in the same file.
`null` is not expected here.

Reply:
Very well, just saying ;)